SmartDeviceLink/SDLSyncMsgVersion.h

```diff
@@ -24,7 +24,7 @@
 - (instancetype)initWithDictionary:(NSMutableDictionary *)dict;
 
 // TODO: (Alex M.)[2016-12-1] Change from NSInteger to UInt8
-- (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion;
+- (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion patchVersion:(NSInteger)patchVersion;
```

Comment:
By altering this API, you've caused a major version change (calls to the old one will now throw a compiler error). You need to deprecate the current one and add the new one. The current one should then just call the new one with "0" as the patch version. Usages of the old one can then be updated to use the new one.